user-manual/import-export/upload-digital-object.rst

```diff
@@ -191,6 +193,207 @@ your :term:`digital object`.
+
+6. If the filename is in hyperlink format, the icon underneath the "Master
+   file", "Reference copy", and "Thumbnail copy" label is coloured black. It 
+   is coloured grey if the user does not have access and the hyperlink is not 
+   shown.
+
+   .. image:: images/digital-object-no-master-access.*
+      :align: center
+      :width: 90%
+      :alt: digital object without access to master representation 
+
+
+   .. TIP::
+
+      Access to the "Master file", "Reference copy", and "Thumbnail copy" can 
+      be configured per user group. The "Access master", "Access reference", 
+      and/or "Access thumbnail" permission can be set to "Grant" or "Deny" for 
+      any user group, including unauthenticated ("anonymous") users. See: :ref:`edit-user-permissions`. 
+
+8. Access to the "Master file", "Reference copy", and "Thumbnail copy" may be
+   overridden by actionable PREMIS rights configured for individual archival 
+   descriptions and their digital object. See :ref:`rights-digital-object`. 
+   For example, if a PREMIS right is added to an archival description where 
+   the "Act" is "Disseminate" and the "Restriction" is "Disallow", then the 
+   hyperlinks in the "Digital object metadata" section are no longer active, 
+   all the document icons are coloured grey, and the reference copy of the 
+   digital object is no longer displayed on the archival description view 
+   page. Instead the customizable text of the "Disallow statement" is
+   displayed at the top of the page as well as in a "Permissions" field for 
+   each Access copy representation, e.g. "Access to this record is restricted 
+   because it contains personal or confidential information. Please contact 
+   the Reference Archivist for more information on accessing this record.". 
+
+   .. image:: images/digital-object-premis-disallow.*
+      :align: center
+      :width: 90%
+      :alt: digital object PREMIS rights set to disallow
+
+   .. NOTE::
+
+      If the "Restriction" were to be changed to "Conditional" and the default 
+      permission settings are enabled then the "Reference copy" and "Thumbnail 
+      copy" become available again as hyperlinks and these representations are 
+      once again displayed on the AtoM templates. The customizable text for 
+      the "Condition Statement" is displayed in a "Permissions" field for the 
+      "Master file" representation, e.g. "This record has not yet been 
+      reviewed for personal or confidential information. Please contact the 
+      Reference Archivist to request access and initiate an access review.".
+
+:ref:`Back to top <upload-digital-object>`
+
+.. _digital-object-preservation:
+
+Digital object metadata for preservation files
+==============================================
+
+If you or another user of a preservation system (e.g. 
+`Archivematica <https://www.archivematica.org/>`_) has sent a 
+:ref:`DIP upload <archivematica:upload-atom:>` to an :term:`archival
```

Comment:
This is throwing a Travis build error, because there's an extra colon at the end (no colon after `atom` in the ref)